app/src/main/res/layout/act_planevent.xml

```diff
@@ -84,14 +93,13 @@
         android:textAlignment="center"
         android:textColor="@color/black"/>
 
-    <EditText
-        android:id="@+id/eventEndTimeEdit"
+    <TextView
+        android:id="@+id/eventEndTimeInput"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_gravity="center"
         android:textSize="20sp"
-        android:ems="10"
-        android:inputType="time"
+        android:textAlignment="center"
```

Comment:
I don't really like the user interface as it is now (unclear user should click and such). I have some ideas how to improve it, would you like to work/brainstorm on it together @elisakaisa or shall I just do it as chief of frontend :P